From bc429808fd1ecb2e1ff6481994e199e2fe23c088 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:24 +0900
Subject: [PATCH 0954/1083] Revert "drm/rcar-du: Fix crash when disabling an
 already disabled plane"

This reverts commit c1b40c31ee49a6e6766e842d917f5afb081afccd.
(cherry picked from commit 5b6ab7b011f4f33ce8dbc10f14607555f4e27ca4)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_plane.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_plane.c b/drivers/gpu/drm/rcar-du/rcar_du_plane.c
index 8cbd293..5055c9c 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_plane.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_plane.c
@@ -311,9 +311,6 @@ static int rcar_du_plane_disable(struct drm_plane *plane)
 	struct rcar_du_device *rcdu = plane->dev->dev_private;
 	struct rcar_du_plane *rplane = to_rcar_plane(plane);
 
-	if (!rplane->enabled)
-		return 0;
-
 	mutex_lock(&rcdu->planes.lock);
 	rplane->enabled = false;
 	rcar_du_crtc_update_planes(rplane->crtc);
-- 
1.8.3.2

